server/services/lduService.js

```diff
@@ -34,13 +35,15 @@ module.exports = function createLduService(deliusClient, activeLduClient) {
         active: activeLdusCodes.includes(ldu.code),
       }))
 
-      const probationArea = {
+      const uniqueLdus = Object.entries(R.groupBy(R.prop('code'), allLdus))
+        .sort((a, b) => a[0].localeCompare(b[0]))
+        .map(entry => entry[1].sort((a, b) => a.description.localeCompare(b.description))[0])
```

Comment:
destructuring and using appropriate variable names to represent what you are comparing might make this a bit clearer: 

```js
 .sort(([lduCode1], [lduCode2]) => lduCode1.localeCompare(lduCode2))
.map([code, ldus]) => ldus.sort((ldu1, ldu2) => ldu1.description.localeCompare(ldu2.description))[0])
```

Reply:
I'm not sure it makes things that much clearer although I think the it's okay to destructure. 

However passing in an array to map([code, ldus]) causes problems with the linter because we don't actually use 'code' , the first item in the array. It's there just because we want to access the second item (descriptions of the ldu), I think ??

But this would work: 
const uniqueLdus = [...Object.entries(R.groupBy(R.prop('code'), allLdus))]
  .sort(([lduCode1], [lduCode2]) => lduCode1.localeCompare(lduCode2))
  .map( ldu => ldu[1].sort((ldu1, ldu2) => ldu1.description.localeCompare(ldu2.description))[0])

Reply:
done